cmd/vicadmin/server.go

```diff
@@ -507,16 +507,45 @@ func (s *server) bundleContainerLogs(res http.ResponseWriter, req *http.Request,
 	s.bundleLogs(res, req, readers, f)
 }
 
+func (s *server) bundleDiagnosticLogs(res http.ResponseWriter, req *http.Request, f format) {
```

Comment:
Please try to merge duplicated codes with bundleContainerLogs.

Reply:
Done